auth/auth/auth.py

```diff
@@ -128,7 +128,7 @@ async def check_valid_new_user(tx: Transaction, username, login_id, is_developer
         raise MultipleUserTypes(username)
     if not is_service_account and not login_id:
         raise EmptyLoginID(username)
-    if not username or not all(c for c in username if c.isalnum()):
+    if not username or not (username.isalnum() and username.islower()):
```

Comment:
Am I missing an important reason why this was being done character by character previously?